test/tmemleak

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+../util/shlib_wrap.sh ./memleaktest freeit || exit 1
```

Comment:
the wrapper script is outdated, i removed it.  the test recipe is "interesting" see the !run :)